INFO  | 03/25/2020 12:49:49 PM | File run_exp.py           | Line 26    | Writing log file to exp/GRAN\GRANMixtureBernoulli_grid_2020-Mar-25-12-49-49_24660\log_exp_24660.txt
INFO  | 03/25/2020 12:49:49 PM | File run_exp.py           | Line 27    | Exp instance id = 24660
INFO  | 03/25/2020 12:49:49 PM | File run_exp.py           | Line 28    | Exp comment = None
INFO  | 03/25/2020 12:49:49 PM | File run_exp.py           | Line 29    | Config =
INFO  | 03/25/2020 12:49:49 PM | File gran_runner.py       | Line 124   | Train/val/test = 80/20/20
INFO  | 03/25/2020 12:49:49 PM | File gran_runner.py       | Line 137   | No Edges vs. Edges in training set = 111.70632737276479
ERROR | 03/25/2020 12:49:51 PM | File run_exp.py           | Line 42    | Traceback (most recent call last):
  File "c:\Users\olive\OneDrive\Dokumenter\GitHub\GRAN\run_exp.py", line 38, in main
    runner.train()
  File "c:\Users\olive\OneDrive\Dokumenter\GitHub\GRAN\runner\gran_runner.py", line 225, in train
    data = train_iterator.next()
  File "C:\Users\olive\Anaconda3\envs\Bachelor\lib\site-packages\torch\utils\data\dataloader.py", line 346, in __next__
    data = self.dataset_fetcher.fetch(index)  # may raise StopIteration
  File "C:\Users\olive\Anaconda3\envs\Bachelor\lib\site-packages\torch\utils\data\_utils\fetch.py", line 44, in fetch
    data = [self.dataset[idx] for idx in possibly_batched_index]
  File "C:\Users\olive\Anaconda3\envs\Bachelor\lib\site-packages\torch\utils\data\_utils\fetch.py", line 44, in <listcomp>
    data = [self.dataset[idx] for idx in possibly_batched_index]
  File "c:\Users\olive\OneDrive\Dokumenter\GitHub\GRAN\dataset\gran_data.py", line 180, in __getitem__
    graph = pickle.load(open(self.file_names[index], 'rb'))
IndexError: list index out of range

